ui/src/app/shared/components/page.tsx

```diff
@@ -51,9 +51,10 @@ export class Page extends React.Component<{title: string; toolbar?: Toolbar | Ob
 
     private async goToLogin(logout = false) {
         if (logout) {
-            await services.users.logout();
+            window.location.href = '/auth/logout';
```

Comment:
Please add `import requests from '../services/requests';` and 

```suggestion
            window.location.href = requests.toAbsURL('/auth/logout');
```

This will append root path to logout URL